metrics/doc.go

```diff
@@ -3,9 +3,26 @@
 // This file is derived from metrics/metrics.go (2018/06/04).
 // See LICENSE in the metrics directory for the original copyright and license.
 
-// Go port of Coda Hale's Metrics library
-//
-// <https://github.com/rcrowley/go-metrics>
-//
-// Coda Hale's original work: <https://github.com/codahale/metrics>
+/*
+Go port of Coda Hale's Metrics library: <https://github.com/rcrowley/go-metrics>
+
+Coda Hale's original work: <https://github.com/codahale/metrics>
+
+Package metrics provides various kinds of metrics which can be used to collect
+system statistics. Collected statistics can be exported via InfluxDB, Librato or
+Prometheus. Different types of metrics have different characteristics.
+
+  - Gauges: hold an int64 value that can be set arbitrarily.
```

Comment:
The explanations described here are the same parts of the type descriptions as the godoc.
![image](https://user-images.githubusercontent.com/13497596/62607364-f3bbf780-b938-11e9-85be-2c680ce25dc9.png)

Reply:
@nogang Yes, I copied the descriptions here as I thought this document is an entry point of the metrics package. They might find descriptions in each file, I thought it would be better to have a place which lists all types of metrics and help the user to find which one to use.